Vagrantfile

```diff
@@ -10,9 +10,10 @@ $SERVER_VERSION= (ENV['SERVER_VERSION'] || $SERVER_VERSION)
 $NETNEXT_SERVER_BOX = (ENV['NETNEXT_SERVER_BOX'] || $NETNEXT_SERVER_BOX)
 $NETNEXT_SERVER_VERSION= (ENV['NETNEXT_SERVER_VERSION'] || $NETNEXT_SERVER_VERSION)
 
-if ENV['NETNEXT'] == "true"
+if ENV['NETNEXT'] == "true" || ENV['NETNEXT'] == "1" then
     $SERVER_BOX = $NETNEXT_SERVER_BOX
     $SERVER_VERSION = $NETNEXT_SERVER_VERSION
+    $vm_kernel = '+'
```

Comment:
This can't be `+` as this symbol isn't acceptable to be part of a k8s node name.

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names

>contain no more than 253 characters
>contain only lowercase alphanumeric characters, ‘-’ or ‘.’
>start with an alphanumeric character
>end with an alphanumeric character

Reply:
Oh, you're right. The hostname should remain `k8s{1,2}`. The `+` should only be used for Vagrant's names, as we currently do for the `build_id_name`. Latest branch update fixes it.